Android/Nothing/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    val kotlin_version = "1.6.10"
+    val kotlin_version = "1.6.0"
```

Comment:
why?

Reply:
Unfortunately, at the moment this PR was created, the compose library/runtime does not support  this version of kotlin. We have to downgrade to keep using the compile time library.

But the same day a new release went out: https://developer.android.com/jetpack/androidx/releases/compose-animation#1.1.0-rc03 

So expect a new PR with the upgrade of both the library and the kotlin version.